lib/Services/executor.js

```diff
@@ -25,11 +25,15 @@ class Executor extends Request {
   /**
    * @param {string} applicationId
    * @param {string} userId
+   * @param {string} ids
+   * @param {string} fields
```

Comment:
Should these not be `string[]`?

Reply:
And since this is a querying related, can this be encapsulated within a `query` object? Just to avoid expanding the signature in case more query parameters are added